Avoid `mips' as a parameter name. It is a reserved name for mips* targets.

--- Cbc/src/CbcModel.hpp	2015-07-07 22:44:40.000000000 +0200
+++ Cbc/src/CbcModel.hpp	2018-01-14 13:49:12.277278333 +0100
@@ -2394,8 +2394,8 @@
     plan10 redoWalkBack();
     //@}
     
-    plan10 setMIPStart( const std::vector< std::pair< std::string, double > > &mips ) {
-       this->mipStart_ = mips;
+    plan10 setMIPStart( const std::vector< std::pair< std::string, double > > &start ) {
+       this->mipStart_ = start;
     }
 
     const std::vector< std::pair< std::string, double > > &getMIPStart() {
